src/pcp/html/switch.html

```diff
@@ -1,7 +1,7 @@
 <div class="fl-switchUI">
-    <span class="flc-switchUI-off fl-switchUI-text"></span>
     <div class="flc-switchUI-control fl-switchUI-control">
         <span class="fl-switchUI-controlKnob"></span>
+        <span class="flc-switchUI-off fl-switchLabel"></span>
```

Comment:
Wouldn't it be better if `fl-switchLabel` is renamed to `fl-switchUILabel` in order to follow the container class naming?